cmd/tempo-vulture/Dockerfile

```diff
@@ -2,4 +2,13 @@ FROM alpine:3.20 as certs
 RUN apk --update add ca-certificates
 ARG TARGETARCH
 COPY bin/linux/tempo-vulture-${TARGETARCH} /tempo-vulture
+
+RUN addgroup -g 10001 -S tempo && \
+    adduser -u 10001 -S tempo -G tempo
+
+RUN mkdir -p /var/tempo && \
```

Comment:
Vulture doesn't write data locally, and so the mkdir and associated chown aren't required.

Reply:
Thanks, fixed.